src/seedu/addressbook/commands/FindCommand.java

```diff
@@ -47,14 +47,26 @@ public CommandResult execute() {
      * @return list of persons found
      */
     private List<ReadOnlyPerson> getPersonsWithNameContainingAnyKeyword(Set<String> keywords) {
+        //Converting keywords to lowercase
+        Set<String> lowercaseKeywords = new HashSet<>();
+        for (String keyword: keywords) {
+            lowercaseKeywords.add(keyword.toLowerCase());
+        }
+
         final List<ReadOnlyPerson> matchedPersons = new ArrayList<>();
         for (ReadOnlyPerson person : addressBook.getAllPersons()) {
-            final Set<String> wordsInName = new HashSet<>(person.getName().getWordsInName());
-            if (!Collections.disjoint(wordsInName, keywords)) {
+            //Convert words in person's name to lowercase
+            Set<String> wordsInName = new HashSet<>();
+            for (String word:person.getName().getWordsInName()) {
```

Comment:
Would be good to follow the format above in line 57 (`for (ReadOnlyPerson person : addressBook.getAllPersons())`). where there are spaces before and after `:`